Sming/Libraries/Adafruit_ILI9341/Adafruit_ILI9341.cpp

```diff
@@ -157,7 +157,8 @@ void Adafruit_ILI9341::begin(void) {
 	transmitCmdData(0xE1, data, 15);    	//Set Gamma
 
 	transmitCmd(0x11);    	//Exit Sleep
-	os_delay_us(120000);
```

Comment:
@avr39-ripe Can you remove that change from the PR. As far as I can see it has nothing to do with SDK 2.1.0.